src/RestControllers/EncounterRestController.php

```diff
@@ -24,6 +24,14 @@ public function __construct()
         $this->encounterService = new EncounterService();
     }
 
+    public function post($pid, $data)
+    {
+        //what  parts of the Encounter fields do I validate
```

Comment:
bare minimum for now:
make sure `date` is Y-m-d in validator
make sure `cat_id` is set to something in validator

Reply:
Is there any docs available for the Validator? that will help now and in the long run

Reply:
The package used for this is particle/validator : 
https://packagist.org/packages/particle/validator

Reply:
pretty nifty things supported by validator documented here:
http://validator.particle-php.com/en/latest/rules/